docs/undergrad.yaml

```diff
@@ -343,10 +343,10 @@ Single Variable Real Analysis:
     antiderivative of a continuous function:
     change of variable:
     integration by parts:
-    generalized integrals:
-    absolutely convergent integrals:
-    integration of asymptotic comparison relationships:
-    semi-convergent integrals:
+    improper integrals:
+    absolute convergence of improper integrals:
+    comparison test for improper integrals:
+    conditional convergence of improper integrals:
```

Comment:
Actually I think that `generalized integrals` was meant to be Riemann integration on something which is not a segment. We could drop that. Semi-convergent integral means something like the limit of the integral from 0 to N exists when N goes to infinity but the the absolute value is not integrable (something like sin(x)/x). I don't about the correct translation.

Reply:
> Actually I think that `generalized integrals` was meant to be Riemann integration on something which is not a segment.

Do you mean something more general than "improper integrals"?  (which for me means integrating over (0,1] as a limit of Riemann-integration over [1/n, 1], or over [1, \infty) as a limit of Riemann-integration over [1, n])

> Semi-convergent integral means something like the limit of the integral from 0 to N exists when N goes to infinity but the the absolute value is not integrable (something like sin(x)/x). I don't about the correct translation.

To me, "conditional convergence of improper integrals" would indicate exactly that.

Reply:
I don't understand why there are two things then. It seems both are more general than Lebesgue integrability since the norm is not necessarily integrable.

Reply:
I looked for an exposition of the list, and found [this](https://www-fourier.ujf-grenoble.fr/~rjoly/Documents/Pedago/MAT302/cours-MAT302-chapitre-integrales-impropres.pdf).  Based on that I might suggest:
```
    improper integrals:
    absolute vs conditional convergence of improper integrals:
    comparison test for improper integrals:
```
as a translation for

>**Intégrales généralisées**
Intégrales absolument convergentes. Intégration des relations de comparaison. Intégrales semi-convergentes.

What do you think?

Reply:
Great!

Reply:
Done.